Atmos Quality of Life

Kyoth25f opened the pull request:
<!--- LICENSE: AGPL -->

## Sobre a PR
Adiciona HFR pra comprar (quando foi a última vez que você viu um?), atualiza o guia de atmosfera, novos itens no RPD e traduções diversas.

<img width="516" height="403" alt="image" src="https://github.com/user-attachments/assets/77a752e1-1d20-41de-8bc9-94ca8b119c35" />

<img width="395" height="329" alt="image" src="https://github.com/user-attachments/assets/6b7023bb-b320-4150-9fc2-bda3424a7335" />

<img width="734" height="154" alt="image" src="https://github.com/user-attachments/assets/258df40d-e751-40bc-95f0-2a90ee704c90" />

<img width="421" height="424" alt="image" src="https://github.com/user-attachments/assets/725e11fc-d88a-4469-89df-862c39fcb450" />

Queria portar também para o RPD imprimir cano colorido. A implementação no funky é completamente diferente, inviável apenas portar. É preciso reescrever, o que não me parece difícil, a lógica aqui.

## Coisas a fazer
- [ ] Terminar de traduzir o guia de gases;
- [x] Permitir que todos os canos, bombas, e similares possam ser construidos nas layers (rpd e menu G); 
- [x] Rever os sprites do cristalizador;
- [x] Fazer o cristalizador ser ancorável e desancorável.

## Changelog

**Changelog**
:cl:
- add: O Reator de Fusão Hypertorus pode ser comprado na cargo! Procure na aba de atmosfera ou pelo nome "rfh".
- add: Adicionados Pipe Manifold e Half Pipe no RPD.
- add: Adicionada aba de sensores e monitores no RPD.
- tweak: Guia de Atmosfera atualizado.
- tweak: O RPD, menu G E cristalizador são compatíveis com o sistema de pipe layers.
- fix: O cristalizador não manterá mais a temperatura com uma quantidade baixa de gás na porta do regulador.
- fix: A taxa máxima de entrada do cristalizador agora está limitada a 250 mol/s.
- fix: O sprite do cristalizador mostra corretamente as duas entradas. Também é possível rotacionar. As duas entradas são diferentes, ainda não há uma forma visual de diferenciar. Se de um lado der errado, tente o outro ou rotacione.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- New Features
  - Added a “Sensors & Monitors” group to construction tools with new build options: air alarm, air sensor, gas pipe sensor, manifold, and half pipe.
  - New cargo order: HFR crate available under Atmospherics.

- Improvements
  - Enhanced multi-layer atmos piping: alignment, alternative variants, and construction paths for pumps, valves, and sensors.

- Balance
  - Reduced HFR console active power usage to 100 kW.

- Localization
  - Brazilian Portuguese updates for construction categories, air alarm, gas pipe sensor, and extensive Guidebook content.

- Documentation
  - Atmospherics guide streamlined; Gases guide translated and expanded.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

marc-pelletier committed:
Update CrystallizerSystem.cs to fix skipping temperature calculations (https://github.com/funky-station/funky-station/pull/760)

marc-pelletier committed:
crystallizer ui tweaks and cap input rate (https://github.com/funky-station/funky-station/pull/799)

Co-authored-by: Steve <marlumpy@gmail.com>

mkanke-real committed:
Adds HFR to cargo order console (https://github.com/funky-station/funky-station/pull/1042)

Terkala committed:
Atmos updates (https://github.com/funky-station/funky-station/pull/1088)

Boaz1111 committed:
More atmos devices can be placed on layers easier. (https://github.com/space-wizards/space-station-14/pull/38672)

marc-pelletier committed:
Lower HFR power draw (https://github.com/funky-station/funky-station/pull/1274)

Co-authored-by: Steve <marlumpy@gmail.com>
Co-authored-by: funkystationbot <funky@funkystation.org>

marc-pelletier committed:
RPD pipe layers (https://github.com/funky-station/funky-station/pull/1338)

Co-authored-by: Steve <marlumpy@gmail.com>
Co-authored-by: funkystationbot <funky@funkystation.org>

Kyoth25f committed:
add: Adiciona traduções

Méo déos... Muita coisa pra traduzir. Não traduzi tudo

Kyoth25f committed:
add: Adiciona canos e sensores no RPD

Kyoth25f committed:
add: adiciona traduções

Cano sensorial é foda

GabyChangelog committed:
chore: Automatically update REUSE headers

GabyChangelog committed:
chore: Automatically update REUSE headers

Kyoth25f commented:
> se nao me engano o funky ja corrigiu isso
Ok, vou atrás.

Mesmo todos os TODO's não estarem completos, esse PR está em bom estado e pode ser mergido.

GabyChangelog commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Kyoth25f commented:
Dois novos objetivos:
- rever os sprites do cristalizador;
- fazer o cristalizador ser ancorável e desancorável.

Kyoth25f committed:
Merge branch 'gaby' into 11-09-2025-atmos-qol

GabyChangelog committed:
chore: Automatically update REUSE headers

Kyoth25f committed:
fix: corrige erro dos ghost vermeio e proto duplicado

Kyoth25f committed:
fix: corrige reuse

Kyoth25f committed:
EDITAR ESSE COMMIT!!

Kyoth25f committed:
fix: resolve tudos os problemas

quase todos...

Kyoth25f committed:
add: adiciona variações de layer

Kyoth25f committed:
add: prototype bulk

GabyChangelog committed:
chore: Automatically update REUSE headers

GabyChangelog committed:
chore: Automatically update REUSE headers

Kyoth25f committed:
fix: remove alts do menu de construção

Kyoth25f committed:
add: alts para air vent

GabyChangelog commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

Kyoth25f committed:
add: adiciona alts para a passive vent

Kyoth25f committed:
fix: corrige cano aparecendo na frente da vent

Kyoth25f committed:
add: adiciona alts dos scrubbers

GabyChangelog committed:
chore: Automatically update REUSE headers

GabyChangelog committed:
chore: Automatically update REUSE headers

Kyoth25f committed:
add: adiciona alts do injetor

Kyoth25f committed:
fix: corrige bug do sprite do conector

Mesmo tudo estando aparentemente correto, ao criar um conector (rpd ou menu G) a textura da porta ficava como ERROR. O
parent define o .rsi que cada pipe layers deveria ter. Como todos os outros binarys tem seus sprites no
pumps.rsi/pumps_alt1.rsi/pumps_alt2.rsi, forçar o .rsi base ser um deles não causa problemas, visto que já é o mesmo.
Mas, no caso do conector, que usa um sprite diferente, quando o .rsi base dele fosse forçado a um dos anteriores,
perdemos a textura correta dele.

A solução é simplesmente descartar o que é definido no parent.

Kyoth25f committed:
tweak: tentativa de remover bolinhas

Isso deveria remover as bolinhas azuis que mostra em construções que permitem pipe layers. Me parece ter algo de errado
com a lógica de mostrar as bolinhas.

Kyoth25f committed:
fix: corrige flip não atualizando

A lógica estava invertida

Kyoth25f committed:
fix: corrige bug das bolinhas azuis

Kyoth25f committed:
fix: corrige mixer flipado errado

GabyChangelog committed:
chore: Automatically update REUSE headers

GabyChangelog committed:
chore: Automatically update REUSE headers

Kyoth25f committed:
add: adiciona alts do inline filter

GabyChangelog committed:
chore: Automatically update REUSE headers

GabyChangelog committed:
chore: Automatically update REUSE headers

Kyoth25f committed:
Merge branch 'gaby' into 11-09-2025-atmos-qol

Kyoth25f committed:
Merge remote-tracking branch 'fork/11-09-2025-atmos-qol' into 11-09-2025-atmos-qol

GabyChangelog committed:
chore: Automatically update REUSE headers

Kyoth25f committed:
fix: yaml fix

Kyoth25f commented:
O funky propositalmente fez o cristalizador não ancorável por balanceamento. Ainda não entendi a problemática e, por isso, não estou alterando aqui.

Esse PR adiciona vários prototypes que não foram traduzidos, em breve usarei dos scripts russos pra resolver esses e vários outros problemas.

Kyoth25f committed:
tweak: muito shit, reescrevi

Kyoth25f committed:
tweak: um cristalizador mais amigável

GabyChangelog committed:
chore: Automatically update REUSE headers
